kubectl-rook-ceph.sh

```diff
@@ -14,7 +14,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-set -eEuo pipefail
+set -exEuo pipefail
```

Comment:
Why is the `-x` needed since this is a golang PR?

Reply:
it was there for testing, removed now.